openproblems/tasks/chromatin_potential/README.md

```diff
@@ -0,0 +1,14 @@
+# Chromatin accessibility prediction
+
+Chromatin accessibility prediction refers to the gene expression prediction of a cell or cell type from ATAC-seq peaks. For a summary or all relevant models, see gene score method in [Jeffrey M. Granja et al.](https://www.biorxiv.org/content/10.1101/2020.04.28.066498v1), [Su Wang et al.](https://pubmed.ncbi.nlm.nih.gov/24263090/) et al.
```

Comment:
Looks like this task assumes Mus Musculus? We should specify if a task only works in a specific species, or otherwise add species information in the dataset.

Reply:
so far this task assumes Mus Musculus. We could add more species later if scicar has datasets from other species.

Reply:
Let's mention that in the README.

Reply:
Sure, I'll add it.